docs/sources/whatsnew/whats-new-in-v9-1.md

```diff
@@ -0,0 +1,188 @@
+
+Now, you can experiment with customization in Grafana's Admin section in the UI, or customize branding and links using the API. This is an early-access feature available only to self-managed customers.
+
+{{< figure src="/static/img/docs/enterprise/custom-branding-9-1.png" max-width="750px" caption="Configuring custom branding" >}}
+
+### Auth and security
+
+#### RBAC for app plugins, usage insights, and query caching
+
+Improvements to RBAC continue. We are rolling out role-based access control (RBAC) across all of Grafana's features. In v9.1, you can determine which users, teams, and roles can access app plugins like OnCall and Synthetics. Note that you can't define view or edit roles for app plugins yet (for example, you cannot yet grant a user view-only access to OnCall). This definition is planned for a future release.
+
+You can also control who can view, edit, or administer dashboard and data source usage insights, as well as data source query caching configuration. For more details, see the [RBAC documentation]({{< relref "../administration/roles-and-permissions/access-control/" >}}).
+
+{{< figure src="/static/img/docs/enterprise/rbac-app-plugins-9-1.png" max-width="750px" caption="Configuring role-based-access to app plugins" >}}
+
+#### Rotate your database encryption keys using Grafana's API
+
+In Grafana version 9.0, we [revamped]({{< relref "./whats-new-in-v9-0/#envelope-encryption-is-generally-available-and-enabled-by-default"  >}}) the method Grafana uses to encrypt secrets, like data source credentials, so that you can rotate encryption keys and integrate with a key management system like Hashicorp Vault, AWS Key Management Service, or Azure Key Vault.
+
+Now, you can rotate keys and re-encrypt secrets via API. This makes it easier to configure Grafana to be secure while deploying it. To learn more, see our guide to [configuring database encryption]({{< relref "../setup-grafana/configure-security/configure-database-encryption/#configure-database-encryption" >}}).
+
+#### Audit all actions in Grafana using Verbose Mode
+
+Auditing logs helps you manage and mitigate activity, and meet compliance requirements. By default, Grafana emits an audit log with every action that changes something (like a user creating or updating a dashboard, or updating another user's permissions). If you want to record all actions on the Grafana server, including GETs and page views, you can now turn on `verbose mode`. This results in more logs, but it can be useful to debug specific issues or make sure you catch everything happening in Grafana for security or compliance reasons. To learn more, see the [configuration documentation]({{< relref "../setup-grafana/configure-grafana/enterprise-configuration/#verbose" >}}).
```

Comment:
```suggestion
Auditing logs helps you manage and mitigate activity and meet compliance requirements. 
By default, Grafana emits an audit log with every action that changes something (like a user creating or updating a dashboard or updating another user's permissions). 
If you want to record all actions on the Grafana server, including GETs and page views, you can now turn on `verbose mode`. 
This results in more logs, but it can be useful to debug specific issues or make sure you catch everything happening in Grafana for security or compliance reasons. 
To learn more, see the [configuration documentation]({{< relref "../setup-grafana/configure-grafana/enterprise-configuration/#verbose" >}}).
```